nifi-nar-bundles/nifi-aws-bundle/nifi-aws-processors/src/test/java/org/apache/nifi/processors/aws/kinesis/stream/ITPutKinesisStreamWithEndpointOverride.java

```diff
@@ -29,24 +31,24 @@
 
 // This integration test can be run against a mock Kenesis such as
 // https://github.com/mhart/kinesalite or https://github.com/localstack/localstack
+@Disabled("Required external service be running. Needs to be updated to make use of Localstack TestContainer")
```

Comment:
I recommend deleting this test class instead of marking it disabled. There is not a lot included, and it would be better to start with a new integration test class.

Reply:
Yeah, fair enough.